### ISSUE TITLE
calling `fit` returns `Status details: Error : Unknown label type: continuous`

### ISSUE THREAD

**Issue by shadiakiki1986** [mljar/mljar-api-python#6]
Hello. When I call `fit` from the mljar python API, it fails with the following status on each of the experiments: `Status details: Error : Unknown label type: continuous`

Edit: also, the `fit` call just hangs without returning any error

**Comment by pplonski** [comment]
can you provide steps to reproduce?

**Comment by shadiakiki1986** [comment]
The steps are just the default `model = Mljar(...)` followed by `model.fit(...)`.
Here is the exact code

```
model = Mljar(
  project='g2-ml take2/ex3',
  #experiment='Get 0.58 A',
  experiment='RF enc-on-raw A',
  metric='auc',
  algorithms=['rfc'],
  validation_kfolds=15,
  validation_shuffle=False,
  validation_stratify=True,
  validation_train_split=None,
  tuning_mode='Sport',
  create_ensemble=True,
  single_algorithm_time_limit='10'
)

model.fit(train_features, train_target)
```

**Comment by shadiakiki1986** [comment]
It seems that this was related to #8 where my target was accidentally continuous whereas the experiment was a binary classification. Feel free to close this issue
